tests/e2e/test_requests.py

```diff
@@ -376,3 +376,27 @@ def request_attributes(method, horizon):
 
     data = np.ndarray(metadata['shape'], metadata['format'], data)
     return metadata, data
+
+
+def gen_sas():
```

Comment:
Assuming that you created that function as you don't like long and nice names, it is still a bad name from where I am standing! :smiling_imp: ~(preferences, preferences...)~
It is too general. There are many functions that generate sas in this file. Why does this one deserve a special place to be called `gen_sas()` and not any of other ones?

If you want to leave it:
- as it can't be local to the test, that function must go to the beginning of the file, so that everyone can see that general function
- be renamed to gen_default_sas() - we assume that these parameters are default
- all places where this function is used already must be updated (separate commit)

Reply:
Sure